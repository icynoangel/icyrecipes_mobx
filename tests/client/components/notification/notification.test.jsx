import React from 'react';
import ReactDOM from 'react-dom';
import Notification from './../../../../src/client/js/components/notification/notification';
import {shallow} from 'enzyme';
import enzymeToJson from 'enzyme-to-json';

describe('<Notification />', function() {
  this.getComponent = () => {
    return <Notification.wrappedComponent ItemsStore={{text: 'test text'}} />;
  };

  it('Should render <Notification />', () => {
    const wrapper = shallow(this.getComponent());
    expect(enzymeToJson(wrapper)).toMatchSnapshot();
  });
});
