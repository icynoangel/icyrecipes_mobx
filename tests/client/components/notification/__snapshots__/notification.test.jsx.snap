// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notification /> Should render <Notification /> 1`] = `
<div
  className="notification"
>
  <div
    className="notification__content"
  >
    test text
  </div>
</div>
`;
